tests/e2e/vsphere.go

```diff
@@ -97,6 +97,9 @@ func (vs *vSphere) queryCNSVolumeSnapshotWithResult(fcdID string,
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// Connect to VC
```

Comment:
Ideally this should be done in `BeforeEach()` or post a destructive step in the test script or test case itself and not here.

Reply:
Added the code here to fix the intermittent failures due to session authentication problems.